src/containers/App.jsx

```diff
@@ -2,26 +2,49 @@ import React, {
   Component,
   StyleSheet,
   Text,
-  View
+  View,
+  DeviceEventEmitter,
+  NativeModules
 } from 'react-native';
+
 import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
 import {locationChanged} from '../redux/modules/location';
 
+const LOSTLocationProvider = NativeModules.LOSTLocationProvider;
 import MapView from '../components/MapView';
 
-export default class App extends Component {
+class App extends Component {
+  componentWillMount() {
+    LOSTLocationProvider.startLocationPolling(500, 0.1, LOSTLocationProvider.HIGH_ACCURACY);
+
```

Comment:
Perhaps there's a refactor here to extract some of the location-provider related stuff? As we create more and more event listeners, I can imagine this file getting pretty hairy... Perhaps we can make an `eventListener` directory and import `locationChangedListener` from it? 
So in our import statements we'd write:

```
import locationChangedListener from "./eventListeners"
```

 Then on line 22, we'd just write:

```
this.locationChangedListener = locationChangedListener
/// and so on for all the event listeners we happen to want at this level...